spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc

```diff
@@ -320,21 +320,7 @@ You can configure the Data Flow server that is on Kubernetes to deploy tasks to
 [[configuration-kubernetes-general]]
 === General Configuration
 
-The Spring Cloud Data Flow server for Kubernetes uses the https://github.com/fabric8io/spring-cloud-kubernetes[`spring-cloud-kubernetes`] module process both the ConfigMap and the secrets settings. To enable the ConfigMap support, pass in an environment variable of `SPRING_CLOUD_KUBERNETES_CONFIG_NAME` and set it to the name of the ConfigMap. The same is true for the secrets, where the environment variable is `SPRING_CLOUD_KUBERNETES_SECRETS_NAME`. To use the secrets, you also need to set `SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API` to `true`.
-
-The following example shows a snippet from a deployment script that sets these environment variables:
-
-
-[source,yaml]
-----
-env:
-- name: SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API
-  value: 'true'
-- name: SPRING_CLOUD_KUBERNETES_SECRETS_NAME
-  value: mysql
-- name: SPRING_CLOUD_KUBERNETES_CONFIG_NAME
-  value: scdf-server
-----
+The Spring Cloud Data Flow server for Kubernetes uses the https://github.com/fabric8io/spring-cloud-kubernetes[`spring-cloud-kubernetes`] module to process secrets that are mounted under `/etc/secrets`. ConfigMaps must be mounted as `application.yaml` in the `/config` directory that is processed by Spring Boot. To avoid access to the Kubernetes API server the `SPRING_CLOUD_KUBERNETES_CONFIG_ENABLE_API` and `SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API` are set to `false`.
```

Comment:
Given these changes aim to remove the usage of `spring-cloud-kubernetes`, I wonder why do we still keep these `SPRING_CLOUD_KUBERNETES_*` settings and set them to `false` instead of removing them completely?

Reply:
or, maybe we still want this to be available until we completely remove the usage (on loading the secrets from mounts to env)?

Reply:
Yes, until Boot supports reading the mounted secrets we need to keep the `spring-cloud-kubernetes`  dependency just to read the secrets from the volume mount.